bookbotproject/goodReadsApi.py

```diff
@@ -0,0 +1,23 @@
+from goodreads import client
+import telegrambot
+
+goodreads_key = "UfEAj6yQFcghrsqrZpQ"
+goodreads_secret = "AgAug5etGSAYceT2SeLGBZLMm803I6LiQSxSnGubOjg"
+gc = client.GoodreadsClient(goodreads_key,goodreads_secret)
+
+
+""" this is a draft code, telegram api, goodreads api and wit ai can send messages among themselves or can make necessary things, for the sake of simplicity we created this py file"""
+
+"""TODO : add bunch of methods like ,search(response): takes response as parameter coming from wit.ai intent and searches .... """
+
+def main():
```

Comment:
As far as I understand, this class will only provide functions for other classes to use, therefore we dont need a main function. You can create a function searchBooks(keyword) and use the "gc.search_books(keyword)" line as the return value.